WireAuthentication/Sources/WireAuthenticationUI/Views/Login/LoginViaEmailOnPrem/LoginViaEmailOnPremView.swift

```diff
@@ -0,0 +1,234 @@
+//
+// Wire
+// Copyright (C) 2025 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import SwiftUI
+import WireDesign
+import WireReusableUIComponents
+
+package protocol LoginViaEmailOnPremViewBuilder {
+
+    @MainActor
+    func loginViaEmailOnPremView(
+        email: String,
+        canCreateAccount: Bool
+    ) -> LoginViaEmailOnPremView
+
+}
+
+package struct LoginViaEmailOnPremView: View {
+    @ObservedObject var viewModel: LoginViaEmailOnPremViewModel
+
+    @State private var password: String = ""
+    @State private var proxyPassword: String = ""
+    @State private var showPasswordRules: Bool = false
+    @State private var showCustomBackendAlert = false
+
+    private var proxyEmail: String = ""
```

Comment:
question: shouldn't this also be @State?

Reply:
I created https://wearezeta.atlassian.net/browse/WPB-16256 to implement Proxy support